make-rules/ips-buildinfo.mk

```diff
@@ -31,7 +31,7 @@
 #
 # Release major number: 2014, 2015, etc.
 #
-RELEASE_MAJOR ?= 2020
+RELEASE_MAJOR ?= 2021
```

Comment:
Should we not do this in a seperate PR?

Reply:
Last time it happened here:
commit c2b8df7766849ddb8f75305e9075721dd400d754
Author: Alexander Pyhalov <apyhalov@gmail.com>
Date:   Tue Jan 14 09:09:45 2020 +0300

    Remove Python2 illumos-gate modules

:100644 100644 8ab8f07f7 c6e71ef85 M    components/meta-packages/history/history
:100644 100644 62f8b7d17 ad7ef8cef M    components/openindiana/illumos-gate/Makefile
:100644 100644 9b1641a60 1f0c43e54 M    make-rules/ips-buildinfo.mk

Reply:
Hmmmm, welp. We should maybe do this properly when the Year increments aswell.... Well let's leave it in then and do it correctly in 2022